build-gluster-org/jobs/user-update.yml

```diff
@@ -0,0 +1,36 @@
+- job:
+    name: update-admin-list
+    node: 
+    description: This job updates the jenkins job's admin-list with the members of gluster-maintainers 
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        url: git@github.com:gluster/build-jobs.git
```

Comment:
This would work only if we have the ssh key to the job. We need it to push, but I am not sure that jenkins is able to use it to pull. I guess the way it should be done is by pulling with https, but pushing on a another remote who is using ssh

(that's the easiest way, the other is to configure a different pushurl , using the trick here: https://git.github.io/git-reference/remotes/#remote-update to have 1 url to pull, 1 to push, so you pull from the main repo, but you push to your fork).

Reply:
I basically used ssh URL and provided credentials of Gluster-ant.
        url: git@github.com:gluster/build-jobs.git
        credentials-id: github-credentials
I have to see the manage permissions of user : Gluster-ant.